### ISSUE TITLE
Question about Hyperparameters for Faster UADA Attack Runtime

### ISSUE THREAD

**Issue by futakw** [William-wAng618/roboticAttack#18]
Thank you for your great work!

Since the UADA attack takes quite a long time, I was wondering if you have tried speeding it up by:

- increasing the learning rate
- increasing the batch size
- reducing the number of iterations
- reducing the inner loop

If so, did any of these work? Also, how sensitive was the performance to these settings?

Thank you in advance!

```shell
python roboticAttack/VLAAttacker/UADA_wrapper.py \
    --maskidx 0 \
    --lr 2e-3 \
    --server $current_dir \
    --device 0 \
    --iter 2000 \
    --accumulate 1 \
    --bs 8 \
    --warmup 20 \
    --tags "debug testrun" \
    --filterGripTrainTo1 false \
    --geometry true \
    --patch_size "3,50,50" \
    --wandb_project "false" \
    --wandb_project "false" \
    --wandb_entity "xxx" \
    --innerLoop 50 \
    --dataset "libero_spatial" # "libero_spatial" / "libero_10" / "libero_goal" / "libero_goal" / "bridge_orig"
```
